examples/healthchecks/README.md

```diff
@@ -82,6 +89,11 @@ and for example here is a filled in database where:
 which when filled out, will look like this
 
 ```python
+INSTALLED_APPS = [
+    'django_spanner',
+    ...
```

Comment:
Leaving three dots in this section which appears under the words "which when filled out" seems misleading.

Reply:
Ack, gotcha, thanks!

Reply:
Removing the dots isn't correct. I figured you would put in all the apps that are needed for the tutorial.

Reply:
Fixed up.